registry/1inch/calldata-AggregationRouterV6.json

```diff
@@ -0,0 +1,495 @@
+{
+    "$schema": "../../specs/erc7730-v1.schema.json",
+
+    "context": {
+        "$id": "AggregationRouterV6",
+        "contract": {
+            "abi": "https://api.etherscan.io/api?module=contract&action=getabi&address=0x111111125421cA6dc452d289314280a0f8842A65",
+            "deployments": [
+                {
+                    "chainId": 1,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
+                },
+                {
+                    "chainId": 10,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
+                },
+                {
+                    "chainId": 56,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
+                },
+                {
+                    "chainId": 100,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
+                },
+                {
+                    "chainId": 137,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
+                },
+                {
+                    "chainId": 250,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
+                },
+                {
+                    "chainId": 324,
+                    "address": "0x111111125421cA6dc452d289314280a0f8842A65"
```

Comment:
On zkSync RouterV6 has different address, correct one is: 0x6fd4383cb451173d5f9304f041c7bcbf27d561ff

Reply:
Is there a way to officially confirm this? I was unaware of the different addresses when I checked a few through etherscan. A repo, doc page, or somewhere else to verify the address?

Reply:
At the moment there is two ways to verify:
1)  it's available on the official Github: https://github.com/1inch/limit-order-protocol
2) You also can perform swap on zkSync on app.1inch.io and see where transaction goes

I also asked 1inch team put it some where into official docs